include/deal.II/numerics/vector_tools_copy_to_different_dof_handler.h

```diff
@@ -0,0 +1,335 @@
+/*
+ * copy_to_different_dof_handler.h
+ *
+ *  Created on: Apr 29, 2020
+ *      Author: mwichro
```

Comment:
Can you put the declaration in include/deal.II/numerics/vector_tools.h,  the implementation in include/deal.II/numerics/vector_tools.templates.h, and the instantiation in a source/numerics/vector_tools_interpolate_to_different_dof_handler.cc and source/numerics/vector_tools_interpolate_to_different_dof_handler.inst.in?

Reply:
You mean `vector_tools_interpolate.h` and `vector_tools_interpolate.templates.h`. :slightly_smiling_face: